tests/semanticdb/expect/Enums.expect.scala

```diff
@@ -12,10 +12,10 @@ object Enums/*<-_empty_::Enums.*/:
     case Hearts/*<-_empty_::Enums.Suits.Hearts.*/, Spades/*<-_empty_::Enums.Suits.Spades.*/, Clubs/*<-_empty_::Enums.Suits.Clubs.*/, Diamonds/*<-_empty_::Enums.Suits.Diamonds.*/
 
   object Suits/*<-_empty_::Enums.Suits.*/:
-    extension (suit/*<-_empty_::Enums.Suits.extension_isRed().(suit)*/: Suits/*->_empty_::Enums.Suits#*/) def isRed: Boolean /*<-_empty_::Enums.Suits.extension_isRed().*//*->scala::Boolean#*/=
+    extension (suit/*<-_empty_::Enums.Suits.extension_isRed().(suit)*/: Suits/*->_empty_::Enums.Suits#*/) def isRed/*<-_empty_::Enums.Suits.extension_isRed().*/: Boolean/*->scala::Boolean#*/ =
       suit/*->_empty_::Enums.Suits.extension_isRed().(suit)*/ ==/*->scala::Any#`==`().*/ Hearts/*->_empty_::Enums.Suits.Hearts.*/ ||/*->scala::Boolean#`||`().*/ suit/*->_empty_::Enums.Suits.extension_isRed().(suit)*/ ==/*->scala::Any#`==`().*/ Diamonds/*->_empty_::Enums.Suits.Diamonds.*/
 
-    extension (suit/*<-_empty_::Enums.Suits.extension_isBlack().(suit)*/: Suits/*->_empty_::Enums.Suits#*/) def isBlack: Boolean /*<-_empty_::Enums.Suits.extension_isBlack().*//*->scala::Boolean#*/= suit/*->_empty_::Enums.Suits.extension_isBlack().(suit)*/ match
+    extension (suit/*<-_empty_::Enums.Suits.extension_isBlack().(suit)*/: Suits/*->_empty_::Enums.Suits#*/) def isBlack/*<-_empty_::Enums.Suits.extension_isBlack().*/: Boolean/*->scala::Boolean#*/ = suit/*->_empty_::Enums.Suits.extension_isBlack().(suit)*/ match
```

Comment:
Sorry, I'm not sure what you mean

Reply:
I am just wondering what is the string representation for extension methods (since basically everything is represented by string in semanticdb)  and can it not conflict with a normal method in the class prefixed with `extension_`
It is highly unlikely that a method like that will be created, but I wonder if we should maybe change the string representation for extension methods.

Reply:
I see, there is a check in desugaring that proves only extension methods can have that prefix, although its probably not been enforced in macros

Reply:
actually it seems val/vars can also have this name 😕

Reply:
what do you think about using `extension-` to signal it's an extension? We can't use `-` in the name so it should be good.

Reply:
so `extension [T,U](t: T) def <*>: (T,U) = ???` would be ```extension-`<*>`().```?

Reply:
the other alternative is that we add another Property kind (alongside FINAL, SEALED, etc.)